pool/tick.gno

```diff
@@ -16,41 +16,23 @@ func tickTickSpacingToMaxLiquidityPerTick(tickSpacing int32) *u256.Uint {
 	maxTick := (consts.MAX_TICK / tickSpacing) * tickSpacing
 	numTicks := uint64((maxTick-minTick)/tickSpacing) + 1
 
-	_maxUint128 := u256.MustFromDecimal(consts.MAX_UINT128)
-	return new(u256.Uint).Div(_maxUint128, u256.NewUint(numTicks))
+	return new(u256.Uint).Div(consts.MAX_UINT128.Clone(), u256.NewUint(numTicks))
```

Comment:
Is `consts.MAX_UINT128` immutable? if not, make it so.

Reply:
332de1e223fa9613049e9eb859b8649cd7c3004f